lacework/resource_lacework_vulnerability_exception_container.go

```diff
@@ -232,6 +241,14 @@ func resourceLaceworkVulnerabilityExceptionContainerCreate(d *schema.ResourceDat
 		}
 	)
 
+	if d.Get("expiry") != nil {
+		expiryTime, err = time.Parse(time.RFC3339, d.Get("expiry").(string))
+		if err != nil {
+			return errors.Wrap(err, fmt.Sprintf("unable to parse expiryTime %s", d.Get("expiry").(string)))
+		}
+	}
+	vulnExCfg.ExpiryTime = expiryTime
```

Comment:
same, we don't use `expiryTime` anywhere else in this function (and other ones), move it to inside the if statement